### ISSUE TITLE
"The document does not contain any accessibility issues" message should be improved

### ISSUE THREAD

**Issue by tiffmartindale** [cksource/ckeditor-plugin-a11ychecker#244]
this implies that the there are no accessibility errors in the document which is something that an automated accessibility checker can't guarantee. It should say something like "No accessibility issues were found."
